cmd/hard/main.go

```diff
@@ -0,0 +1,280 @@
+	if _, err := exec.LookPath("docker"); err != nil {
+		fmt.Println("ERROR: Docker is not installed or not in PATH")
+		os.Exit(1)
+	}
+
+	if _, err := exec.LookPath("docker-compose"); err == nil {
+		return exec.Command("docker-compose", "-f", dockerComposePath)
+	}
+
+	return exec.Command("docker", "compose", "-f", dockerComposePath)
+}
+
+var upCmd = &cobra.Command{
+	Use:   "up",
+	Short: "Start the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "up")
+		if detach, _ := cmd.Flags().GetBool("detach"); detach {
+			dcCmd.Args = append(dcCmd.Args, "-d")
+		}
+		dcCmd.Args = append(dcCmd.Args, args...)
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var downCmd = &cobra.Command{
+	Use:   "down",
+	Short: "Stop the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "down")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var restartCmd = &cobra.Command{
+	Use:   "restart",
+	Short: "Restart the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "restart")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var psCmd = &cobra.Command{
+	Use:   "ps",
+	Short: "Display the status of all containers",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "ps")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var bashCmd = &cobra.Command{
+	Use:   "bash",
+	Short: "Start a shell session within the app container",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "exec", "app", "bash")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Stdin = os.Stdin
+		dcCmd.Run()
+	},
+}
+
+var laravelCmd = &cobra.Command{
+	Use:   "laravel",
+	Short: "Run Laravel commands",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "exec", "app", "laravel")
+		dcCmd.Args = append(dcCmd.Args, args...)
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Stdin = os.Stdin
+		dcCmd.Run()
+	},
+}
+
+func runInProject(projectCmd string, args []string) {
+	project := ""
+	if len(args) > 0 {
+		if _, err := os.Stat(filepath.Join(wwwPath, args[0])); err == nil {
+			project = args[0]
+			args = args[1:]
+		}
+	}
+
+	if project == "" {
+		dir, _ := os.Getwd()
+		project = filepath.Base(dir)
+	}
+
+	dcCmd := dockerComposeCmd()
+	dcCmd.Args = append(dcCmd.Args, "exec", "-it", "--workdir", "/var/www/"+project, "app", projectCmd)
+	dcCmd.Args = append(dcCmd.Args, args...)
+	dcCmd.Stdout = os.Stdout
+	dcCmd.Stderr = os.Stderr
+	dcCmd.Stdin = os.Stdin
+	dcCmd.Run()
+}
+
+var phpCmd = &cobra.Command{
```

Comment:
Not working properly. Hard cli don't what to do with "-" flag

![image](https://github.com/user-attachments/assets/a7b2adc1-2096-43ba-b39d-dd6f4f6f9065)

Reply:
Now all command flags are passed to the command within docker